Content.Server/Nutrition/EntitySystems/FoodSystem.cs

```diff
@@ -322,6 +323,10 @@ public void DeleteAndSpawnTrash(FoodComponent component, EntityUid food, EntityU
         if (ev.Cancelled)
             return;
 
+        var dev = new DestructionEventArgs();
```

Comment:
what food needs this???

Reply:
Yeah it technically isn't needed right now but I want redo https://github.com/space-wizards/space-station-14/pull/25306 and It's necessary for that. I don't think combining that with this PR make sense. Should I just remove this for now and re-add it in when I remake the other PR?

Reply:
fair enough

Reply:
I would probably move this to a separate PR. It could be useful for more things than this, for instance:

> what food needs this???

Jackboots for one. Moths can eat some crazy things.